apps/core/scenes/profile/Profile.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+
+import * as React from 'react';
+import { ScrollView } from 'react-native';
+import { StyleSheet, Button, Card } from '@kiwicom/universal-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { margaritaTokens } from '@kiwicom/margarita-design-tokens';
+import { Illustration, Text } from '@kiwicom/margarita-components';
+
+type Props = {||};
+
+export default class Profile extends React.Component<Props> {
+  handleSignIn = () => {
+    // @TODO
+  };
+
+  render() {
+    return (
+      <ScrollView
+        style={styles.container}
+        contentContainerStyle={styles.containerContent}
+      >
+        <Illustration name="Lounge" style={styles.illustration} />
+        <Card style={styles.card}>
+          <Text size="large" type="primary" weight="bold">
+            Sign In
+          </Text>
+          <Text size="small" type="secondary" style={styles.description}>
+            Sign in to see your bookings and tickets
+          </Text>
+          <Button onPress={this.handleSignIn} label="Sign In" />
+        </Card>
+      </ScrollView>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: defaultTokens.backgroundBody,
+  },
+  containerContent: {
+    alignItems: 'center',
+  },
+  illustration: {
+    paddingTop: parseInt(defaultTokens.spaceMedium, 10),
+    paddingBottom: parseInt(defaultTokens.spaceLarge, 10),
+  },
+  card: {
+    width: '100%',
+    web: {
```

Comment:
What about using new `ContentContainer` component instead of this?

Reply:
True, thanks, I forget that we already have it. Replaced

Reply:
OK, but I think you don't need this style anymore. `maxWidth` should be handled by `ContentContainer`

Reply:
👍 Right, unnecessary style removed.